blog.html

```diff
@@ -22,6 +22,160 @@
+                    features and improvements. This blog post will highlight some of the more prominent ones.
+                    For a full list of changes, be sure to check the <a href="https://download.apache.org/dist/kafka/4.1.0/RELEASE_NOTES.html">release notes</a>. 
+                </p>
+                <p>
+                    Queues for Kafka (<a href="https://cwiki.apache.org/confluence/x/4hA0Dw">KIP-932</a>) is now in preview. It's still not ready for 
+                    production but you can start evaluating and testing it. See the 
+                    <a href="https://cwiki.apache.org/confluence/x/CIq3FQ">preview release notes</a>
+                    for more details.
+                </p>
+                <p>
+                    This release also introduces a new Streams Rebalance Protocol (<a href="https://cwiki.apache.org/confluence/x/2BCTEg">KIP-1071</a>)
+                    in early access. It is based on the new consumer group protocol (<a href="https://cwiki.apache.org/confluence/x/HhD1D">KIP-848</a>).
+                <p>
+                    See the <a href="https://kafka.apache.org/documentation.html#upgrade_4_1_0">Upgrading to 4.1</a> section in the
+                    documentation for the list of notable changes and detailed upgrade steps.
+                </p>
+                
+                <h3>Kafka Broker, Controller, Producer, Consumer and Admin Client</h3>
+
+                <ul>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/lY3GDQ">KIP-877: Mechanism for plugins and connectors to register metrics</a>
+                        <br>
+                        Many client-side plugins can now implement the <code>Monitorable</code> interface to easily register their own metrics. Tags identifying
+                        the plugin are automatically injected and the metrics use the <code>kafka.CLIENT:type=plugins</code> naming where CLIENT is either producer, consumer or admin.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/8ItyEg">KIP-1050: Consistent error handling for Transactions</a>
+                        <br>
+                        This KIP updates the error handling logic and documentation of all the transaction APIs to make it simpler to build robust applications and build
+                        third party Kafka clients that behave the same way as the Java client.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/JQstEw">KIP-1092: Extend Consumer#close with an option to leave the group or not</a>
+                        <br>
+                        This adds a new <code>Consumer.close(CloseOptions)</code> method which let's define if the consumer should explicitly leave its group when
```

Comment:
```suggestion
                        This adds a new <code>Consumer.close(CloseOptions)</code> method which lets us define if the consumer should explicitly leave its group when
```

nit: I think "lets us" might read a bit more smoothly here?

Reply:
Yes that's a typo, fixed, thanks